src/restic/progress.go

```diff
@@ -43,7 +43,6 @@ type Stat struct {
 	Errors uint64
 }
 
-// ProgressFunc is used to report progress back to the user.
```

Comment:
it may have been an accident. 
I am not well versed with git. if you wish to move it over to new git repo layout you may go ahead.